Updates to topics for MW UI changes. (#427)
Concordium/concordium.github.io: source/shared/net/mobile-wallet/make-shielded-transfer-mw.rst

#. Go to the Accounts page.

#. Tap the Account you want to transfer CCD from or tap |moredetails| for the account.

   - If shielding is hidden for the account, go to the |hamburger| menu in the upper right corner and tap **Show shielded balance for account name**.

#. Tap the **Shielded balance** area of the account card.

#. Tap **SEND**.

#. Enter the amount you want to transfer.

> #. In **Paste receipient address**, enter the receipient address for the receipient of the transfer.

Reviewer: ```suggestion
#. In **Paste recipient address**, enter the recipient address for the recipient of the transfer.
```

>    - You can either choose a recipient that already exists in your **Address Book**, or add a new one by tapping  **Scan QR**.

Reviewer: ```suggestion
   - You can also choose a recipient that already exists in your **Address Book**, or add a new one by tapping  **Scan QR**.
```

#. Go to the Accounts page.

#. Tap the Account you want to transfer CCD from or tap |moredetails| for the account.

   - If shielding is hidden for the account, go to the |hamburger| menu in the upper right corner and tap **Show shielded balance for account name**.

> #. Tap the **Shielded balance** area of the account card.

Reviewer: ```suggestion
#. Tap the **Shielded balance** tab of the totals area.
```
